src/main/java/org/broadinstitute/barclay/argparser/LegacyCommandLineArgumentParser.java

```diff
@@ -423,13 +423,16 @@ private boolean checkNumArguments() {
                 }
                 if (optionDefinition.isCollection) {
                     final Collection<?> c = (Collection<?>) optionDefinition.field.get(optionDefinition.parent);
+                    if (c.size() == 0 && !optionDefinition.optional) {
+                        messageStream.print("ERROR: Option '" + fullName + "' is required");
```

Comment:
instead of `c.size() == 0`, use `c.isEmpty()`

Reply:
Fixed.